pg_spot_operator/cloud_impl/aws_vm.py

```diff
@@ -851,7 +852,7 @@ def ensure_spot_vm(
     if m.vm.storage_type == STORAGE_TYPE_NETWORK:
         vol_desc = ensure_volume_attached(m, i_desc)
 
-    if m.assign_public_ip and not m.floating_public_ip:
+    if m.assign_public_ip and not m.floating_ips:
```

Comment:
This is exactly why I'm thinking the naming should be like `use/assign/something_floating_ips`; it seems to rhyme with `assign_public_ip`